tests/unit/button/button_core_deprecated.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * button_core.js
+ */
+
+
+(function($) {
+
+module( "Button: core deprecated" );
+
+test( "Calling button on a checkbox input calls checkboxradio widget", function(){
+	var checkbox = $( "#checkbox01" );
+
+	expect( 2 );
```

Comment:
@supergray This is extremely unhelpful. We have a style guide that requires semicolons. Commenting on a random commit about the use of semicolons is unproductive. Please try to refrain from such comments in the future.